webapp/src/ts/modules/reports/reports.component.ts

```diff
@@ -468,7 +469,7 @@ export class ReportsComponent implements OnInit, AfterViewInit, OnDestroy {
     if (this.areAllReportsSelected()) {
       return;
     }
-
+    this.isSelectingAllReports =true
```

Comment:
Please fix linting here: 
```suggestion
    this.isSelectingAllReports = true;
```